modules/launcher/src/main/java/org/mule/module/launcher/application/DefaultMuleApplication.java

```diff
@@ -171,14 +171,13 @@ public void init()
             if (!cfgBuilder.isConfigured())
             {
                 List<ConfigurationBuilder> builders = new LinkedList<>();
+                builders.add(cfgBuilder);
```

Comment:
i don't remember exactly. It had something to do spring now having some precedence. Not sure to be honest